### ISSUE TITLE
Bone marrow edema

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10751]
**Preferred term label:**
Bone marrow edema

**Synonyms**

**Definition (free text, please give PubMed ID)**
MRI showing inflammation in the bone marrow; trabecular edema

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Ulnocarpal abutment syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
[Bone marrow edema HP:6000917](https://hpo.jax.org/browse/term/HP:6000917)
